flask/backend.Dockerfile

```diff
@@ -0,0 +1,31 @@
+# # This is a multi-stage build, so we can use the same Dockerfile for dev and prod environments
+# Prod environment
+FROM --platform=$BUILDPLATFORM python:3.10-alpine AS builder
+
+WORKDIR /src
+COPY requirements.txt /src
+RUN --mount=type=cache,target=/root/.cache/pip \
+    pip3 install -r requirements.txt
+
+COPY . .
+
+CMD ["python3", "server.py"]
+
+
+# Dev environment 
```

Comment:
Is this required @henilp105  ? If not, we can consider removing this and make this PR clean so that we can merge it in next few days.

Reply:
Sure.